### ISSUE TITLE
how clear all work  when destroying the viewer 

### ISSUE THREAD

**Issue by jonSeetrue** [Kitware/itk-vtk-viewer#628]
i am a having an memory issue when the viewer is been  destroy form the Dom the   web-worker  are still renaming  and when  the new viewer is been  crated  it dose not clear the old one's so if will load and destroy the viewer  few time eventuality  i will  have a memory issue.
is there a method of terminate the worker before destroying the viewer?

**Comment by PaulHax** [comment]
Good point.  We can call `worker.terminate()` on the WebworkerPromises.  

Workers are left hanging in a few module globals and elsewhere. 
https://github.com/Kitware/itk-vtk-viewer/blob/bbfe8a00c72d787b98747e109c29b3cb068203c7/src/IO/MultiscaleSpatialImage.js#L12-L15

Do you have the time to make a PR fix these?

**Comment by jonSeetrue** [comment]
Hi
i will be glad.
will try to work on it  and will let you know.
